dashboard/modules/metrics/metrics_head.py

```diff
@@ -217,9 +222,71 @@ def _create_default_prometheus_configs(self):
         os.makedirs(os.path.dirname(prometheus_config_output_path), exist_ok=True)
         shutil.copy(PROMETHEUS_CONFIG_INPUT_PATH, prometheus_config_output_path)
 
+    async def _check_grafana_running(self) -> bool:
```

Comment:
should we reuse the code from `grafana_health `?

Reply:
Is there a similar `prometheus_health`?

Reply:
the logic is close but divergent enough where it would be difficult to re-use without a bunch of boilerplate.